docs/index.rst

```diff
@@ -54,34 +47,55 @@ Usage
         time.sleep(1)
 
 
-API Documentation
------------------
+More :doc:`examples`
 
-If you are looking for information on a specific function, class, or method,
-this part of the documentation is for you.
+When **not** to use Schedule
+----------------------------
+Let's be honest, Schedule is not a 'one size fits all' scheduling library.
+This library is designed to be a simple solution for simple scheduling problems.
+You should probably look somewhere else if you need:
 
-.. toctree::
-   api
+* Job persistence (remember schedule between restarts)
+* Exact timing (sub-second precision execution)
+* Concurrent execution (multiple threads)
+* Localization (timezones, workdays or holidays)
 
-Common Questions
```

Comment:
Maybe keep a separate heading for the FAQ with a link to the page?
Many people will be looking for this page.